model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -759,6 +757,8 @@ case class AzureManagedAppCoordinates(tenantId: UUID,
                                       landingZoneId: Option[UUID] = None
 )
 
+case class GcpProjectInfo(projectId: GoogleProjectId)
```

Comment:
Does this merit publishing a new `rawls-model`?

Reply:
I think this is only being consumed internally by the AggregatedWorkspaceService, so I'd say it's not necessary.